docs/manual/java/releases/Migration16.md

```diff
@@ -156,3 +156,9 @@ The setup above makes all traffic and durable data produced by Lagom 1.6 code to
 #### Step 2: new binary, default serializers
 
 Once all your production nodes are running a binary of your service based on Lagom 1.6 you can remove the settings for the old serializers from `application.conf` and use the defaults.
+
+## Known issues
```

Comment:
I don't think this section title is correct. It gives the impression that Lagom is shipped with this bug in `1.6.3` (which is the version when this section was added when, in fact, that's the version when it was removed). Also, there are other known bugs not listed here.

I think we need a new row on the [upgrading table](https://www.lagomframework.com/documentation/1.6.x/scala/Migration16.html#A-note-on-Rolling-Updates-and-Versions) and we could indicate the reason why the step upgrade is necessary (e.g. CBOR) and include the x-link to akka docs. That is, move the paragraph below next to the [upgrading table](https://www.lagomframework.com/documentation/1.6.x/scala/Migration16.html#A-note-on-Rolling-Updates-and-Versions).

Reply:
I moved the paragraph. But updating table already has a row with `1.6.3` and I don't understand what more is needed 🤷‍♂️